npe2/manifest/schema.py

```diff
@@ -101,16 +102,25 @@ class PluginManifest(BaseModel):
     # the actual mechanism/consumption of plugin information) independently
     # of napari itself
 
-    # TODO: refactor entry_point to binding points for activate,deactivate
-    # TODO: Point to activate function
-    # TODO: Point to deactivate function
-
-    # The module that has the activate() function
-    entry_point: Optional[str] = Field(
+    activate_function: Optional[str] = Field(
```

Comment:
I'm not a fan of the "function" in the name.  On suggestion might be:

```suggestion
    on_activate: Optional[str] = Field(
```

Reply:
thanks, I was also so-so on that.  I considered `on_activate` in my head too, so that's an easy one

Reply:
renamed